docs/gen-ai/gen-ai-metrics.md

```diff
@@ -2,28 +2,32 @@
 linkTitle: Generative AI metrics
 --->
 
-# Semantic Conventions for Generative AI Client Metrics
+# Semantic Conventions for Generative AI Metrics
 
 **Status**: [Experimental][DocumentStatus]
 
-The conventions described in this section are specific to Generative AI client
-applications.
-
-**Disclaimer:** These are initial Generative AI client metric instruments
-and attributes but more may be added in the future.
-
 <!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
 
 <!-- toc -->
 
 - [Generative AI Client Metrics](#generative-ai-client-metrics)
   - [Metric: `gen_ai.client.token.usage`](#metric-gen_aiclienttokenusage)
   - [Metric: `gen_ai.client.operation.duration`](#metric-gen_aiclientoperationduration)
+- [Generative AI Model Server Metrics](#generative-ai-model-server-metrics)
+  - [Metric: `gen_ai.server.latency.request_duration`](#metric-gen_aiserverlatencyrequest_duration)
+  - [Metric: `gen_ai.server.latency.time_per_output_token`](#metric-gen_aiserverlatencytime_per_output_token)
+  - [Metric: `gen_ai.server.latency.time_to_first_token`](#metric-gen_aiserverlatencytime_to_first_token)
```

Comment:
why not `gen_ai.server.time_to_first_token` ? `latency` seems redundant

Reply:
The reason to add latency is to group model server metrics into different categories like `latency`, `capacity`, `admission`, etc. But I'm okay with not having the grouping. I've updated the names.